common/workunit/workunit.cpp

```diff
@@ -1087,6 +1089,17 @@ class CDaliWorkUnit : public CPersistedWorkUnit
         VStringBuffer path("/GraphProgress/%s/%s", queryWuid(), graphName);
         return querySDS().connect(path, myProcessSession(), RTM_LOCK_WRITE|RTM_CREATE_QUERY, SDS_LOCK_TIMEOUT);
     }
+    IPropertyTree *getGraphProgressTree() const
+    {
+        IRemoteConnection *conn = queryProgressConnection();
```

Comment:
That COULD be an issue (for thor reading the subgraph status...) Should I call conn->reload or something?